packages/db-mongodb/src/queries/buildSearchParams.ts

```diff
@@ -187,7 +187,7 @@ export async function buildSearchParam({
       return relationshipQuery
     }
 
-    if (formattedOperator && validOperators.includes(formattedOperator as Operator)) {
+    if (formattedOperator && validOperatorMap[formattedOperator as Operator]) {
```

Comment:
You could add this if you want https://github.com/payloadcms/payload/pull/10609#issuecomment-2595862268